components/HomeSection.vue

```diff
@@ -95,6 +95,7 @@ export default Vue.extend({
         break;
       }
       case 'upnext': {
+        console.error('We getting here');
```

Comment:
what??? lol :p

Reply:
Haha, yeah, I was trying to track down why the Up Next section was merged with the first Latest section on my server. So I wanted to see if we were getting to that case lol

Reply:
ha I do the same thing lol